### DISCUSSION TITLE
Relative path problem

### DISCUSSION THREAD

**Discussion by Gahotx** [vercel/vercel#6875] [Help] [answered]
The relative path in vercel will automatically omit the name of the current folder, resulting in the website content not being loaded, as shown in the figure below：
![image](https://user-images.githubusercontent.com/53534938/138379336-a352ff96-6bec-45db-9b2b-dbd9215ca6c3.png)
![image](https://user-images.githubusercontent.com/53534938/138379407-e561979b-762c-4edf-9be4-133ce978dc5c.png)
This happens when accessing through a custom domain name in vercel, but there is no problem accessing through the github URL. I don’t know if this is a bug. The current method can only be to write the folder of the root directory when writing the path. For example, the'study' in the example is also written ('study/img/..'), so that the vercel display is no problem, but my local There will be problems when previewing. (Translated directly from Google Translate, bear with me)

**Comment by Gahotx** [comment, marked as answer]
I found that when the URL is written in full'/study/index.html', Vercel will recognize the path study, but not in full, that is,'/study'. The starting path of the file is'/' instead of'/study'
![image](https://user-images.githubusercontent.com/53534938/138384781-e1da1963-ce64-40c7-814d-b3632abe16ea.png)

**Comment by Levelleor** [comment]
Any idea why this behavior is like this? I don't understand why in this case /study won't refer to relative paths properly? 

I experienced the same issue with bunch of smaller projects being located under public directory all referring relatively to paths within it but it doesn't work correctly unless I call index.html directly which is fine unless someone forgets to go to index.html directly and just calls a "/study/" and find themselves out at a page without styles at all.
